packages/blue-sdk/src/position/Position.ts

```diff
@@ -166,18 +172,10 @@ export class AccrualPosition extends Position implements IAccrualPosition {
     return this.market.getBorrowCapacityUsage(this);
   }
 
-  get borrowCapacityLimit() {
-    return this.market.getBorrowCapacityLimit(this);
-  }
-
   get withdrawCapacityLimit() {
     return this.market.getWithdrawCapacityLimit(this);
   }
 
-  get withdrawCollateralCapacityLimit() {
```

Comment:
This is a breaking change, don't we want to leave it as deprecated for now and remove it in the next major release?